src/dev/create-extension.ts

```diff
@@ -150,15 +210,16 @@ const main = async (rawArgs: string[]) => {
     prettyLog.info(`Extension name: ${projectName}\n`);
 
     prettyLog.info("Getting list of changed files...", 1);
-    const changedFiles = await getChangedFilesFromFirstCommit(projectPath);
+    const changedFiles = await getDiffFilesFromFirstCommit({ projectPath });
+    const deletedFiles = await getDiffFilesFromFirstCommit({ projectPath, isDeleted: true });
 
-    if (changedFiles.length === 0) {
+    if (changedFiles.length === 0 && deletedFiles.length === 0) {
       prettyLog.warning("No changed files to copy.", 1);
       console.log("\n");
     } else {
       prettyLog.info(`Found ${changedFiles.length} changed files, processing them...`, 1);
       console.log("\n");
-      await copyFiles(changedFiles, projectName, projectPath, templates);
+      await copyFiles(changedFiles, deletedFiles, projectName, projectPath, templates);
```

Comment:
and probably we need to rename the function to `copyChanges`

Reply:
Right yes updated 🙌